src/js/view/controls/controlbar.js

```diff
@@ -350,8 +358,12 @@ export default class Controlbar {
         const volumeTooltip = this.elements.volumetooltip;
         // mute, volume, and volumetooltip do not exist on mobile devices.
         if (mute) {
-            toggleClass(mute.element(), 'jw-off', muted);
-            toggleClass(mute.element(), 'jw-full', !muted);
+            const muteElement = mute.element();
+            toggleClass(muteElement, 'jw-off', muted);
+            toggleClass(muteElement, 'jw-full', !muted);
+
+            const ariaText = muted ? this._model.get('localization').unmute : 'Volume';
```

Comment:
we want to move away from hard-coding. @aliciahurst  It seems like we may need to introduce a `mute` key.

Reply:
Yes, we do, and we will. But I forgot to do that in this pass, so we'll have to add it later.